Testscripts/Linux/LSVMBUS.sh

```diff
@@ -147,48 +148,51 @@ else
     expected_network_counter=$VCPU
 fi
 
-case $VM_Size in
-    # If Lv2, the cpu count that attached to the SCSI driver is MIN(the number of vCPUs, 64).
-    *Standard_L*s_v2*)
-        if [ "$VCPU" -gt 64 ];then
-            expected_scsi_counter=64
-        else
-            expected_scsi_counter=$VCPU
-        fi
-    ;;
-    # Default: the cpu count that attached to the SCSI driver is MIN(the number of vCPUs/4, 64).
-    *)
-        if [ "$VCPU" -gt $((64*4)) ];then
-            expected_scsi_counter=64
-        else
-            expected_scsi_counter=$(bc <<<"scale=2;$VCPU/4")
-            if ! [[ "$expected_scsi_counter" =~ \.00 ]]; then
-                # In this case we have a float number that needs to be rounded up.
-                # For example with 6 cores, scsi controller is spread on 2 cores.
-                expected_scsi_counter=$(bc <<<"("$expected_scsi_counter"+1)/1")
-            fi
-            # normalizing the number to integer
-            expected_scsi_counter=${expected_scsi_counter%.*}
-        fi
-    ;;
-esac
+# Default: the cpu count that attached to the SCSI driver is MIN(the number of vCPUs/4, 64).
+if [ "$VCPU" -gt $((64*4)) ];then
+    expected_scsi_counter=64
+else
+    expected_scsi_counter=$(bc <<<"scale=2;$VCPU/4")
+    if ! [[ "$expected_scsi_counter" =~ \.00 ]]; then
+        # In this case we have a float number that needs to be rounded up.
+        # For example with 6 cores, scsi controller is spread on 2 cores.
+        expected_scsi_counter=$(bc <<<"("$expected_scsi_counter"+1)/1")
+    fi
+    # normalizing the number to integer
+    expected_scsi_counter=${expected_scsi_counter%.*}
+fi
+
+testfail=0
+for adapter in "${!ADAPTER_DICT[@]}"; do
+    if [ "${ADAPTER_DICT[$adapter]}" != "$expected_network_counter" ]; then
+        error_msg="Error: values are wrong. Expected for network adapter $adapter channels: $expected_network_counter and actual: ${ADAPTER_DICT[$adapter]}"
```

Comment:
could we use the real name, instead of an 'index' number here? is it possible to make messages like this:  
Expected channels for network adapter $adapterName: ...... and actual: ....